extension/my-style.js

```diff
@@ -35,6 +35,11 @@
     };
   }
 
+  /* Store CSS per hostname, similar to the scoping of localStorage. */
+  function storageKey() {
```

Comment:
Could you add a verb to this function name to be consistent? e.g. `getStorageKey()`